votor/src/event_handler.rs

```diff
@@ -164,7 +166,9 @@ impl EventHandler {
             let mut send_votes_batch_time = Measure::start("send_votes_batch");
             for vote in votes {
                 local_context.stats.incr_vote(&vote);
-                vctx.bls_sender.send(vote).map_err(|_| SendError(()))?;
+                vctx.bls_sender
```

Comment:
should this be in a separate PR?

Reply:
Err, we need this here because we are changing SenderDisconnected from "coverting from SendError" to accept either SendError or TrySendError

Reply:
got it.